admin-dev/themes/new-theme/js/pages/product/components/combination-modal/index.ts

```diff
@@ -28,18 +28,21 @@ import VueI18n from 'vue-i18n';
 import EventEmitter from '@components/event-emitter';
 import ReplaceFormatter from '@vue/plugins/vue-i18n/replace-formatter';
 import CombinationModal from '@pages/product/components/combination-modal/CombinationModal.vue';
+import PaginatedCombinationsService from '@pages/product/services/paginated-combinations-service';
 
 Vue.use(VueI18n);
 
 /**
  * @param {string} combinationModalSelector
+ * @param {PaginatedCombinationsService} paginatedCombinationsService
  * @param {int} productId
  * @param {Object} eventEmitter
  *
  * @returns {Vue|CombinedVueInstance<Vue, {eventEmitter, productId}, object, object, Record<never, any>>|null}
  */
 export default function initCombinationModal(
   combinationModalSelector: string,
+  paginatedCombinationsService: PaginatedCombinationsService,
   productId: number,
```

Comment:
```suggestion
```

Reply:
To check but it could be removed since now it's handled internally by the `paginatedCombinationsService`